PRIVACYROADMAP.MD

```diff
@@ -0,0 +1,26 @@
+# Privacy Roadmap
+
+This document provides a more detailed view of the privacy roadmap. Like [the overall roadmap](ROADMAP.md), 
+it is a living document, which will evolve and change over time.  In particular the features in later versions 
+are likely to be refined and change.
+
+We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  
+Our now scale is about 3 months, next about 6 months, and then later is beyond.
+
+## Now (up to v1.2)
+
+* Ability to add and remove members from privacy groups
```

Comment:
Can we move the membership changes to the top of the next.  I think the work will be complicated enough in implementing that it won't make 1.2.

Reply:
Done